server/channels/store/sqlstore/access_control_policy_store.go

```diff
@@ -381,6 +381,85 @@ func (s *SqlAccessControlPolicyStore) SetActiveStatus(rctx request.CTX, id strin
 	return existingPolicy, nil
 }
 
+func (s *SqlAccessControlPolicyStore) SetActiveStatusMultiple(rctx request.CTX, list []model.AccessControlPolicyActiveUpdate) ([]*model.AccessControlPolicy, error) {
```

Comment:
so, will this always deal with channel policies? I ask because if it at some point deals with parent policies, should it then cascade down as the current setActive (line 359) does?